### PR TITLE
[ethjs] Add cljsjs/ethjs "0.3.0-0"

### PR DESCRIPTION
**Pull request by craig-ludington**
* Ethjs is a highly optimized, light-weight JS utility for Ethereum.

### CODE REVIEW

File: ethjs/build.boot [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  pom {:project     'cljsjs/ethjs
+       :version     +version+
+       :description "A highly optimised, light-weight JS utility for Ethereum."
+       :url         "https://github.com/ethjs/ethjs"
+       :scm         {:url "https://github.com/ethjs/ethjs"}
+       :license     {"MIT" "https://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+   (download :url "https://cdn.jsdelivr.net/npm/ethjs@0.3.0/dist/ethjs.js")
```
Comment (Deraen) [inline ethjs/build.boot]: Please use `+lib-version+` to build the URL here (`format` is good choice).

File: ethjs/build.boot [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   (download :url "https://cdn.jsdelivr.net/npm/ethjs@0.3.0/dist/ethjs.min.js")
+   (sift :move {#".*ethjs.js"        "cljsjs/ethjs/development/ethjs.js"
+                #".*ethjs.min.js"    "cljsjs/ethjs/production/ethjs.min.js"})
+   (sift :include #{#"^cljsjs" #"^deps.cljs"})
```
Comment (Deraen) [inline ethjs/build.boot]: It is better to use `deps-cljs` task to generate `deps.cljs` than to include hand written file. The task will ensure the paths are correct etc.

File: ethjs/build.boot [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+   (pom)
+   (jar)))
```
Comment (Deraen) [inline ethjs/build.boot]: You should add `validate` task as last call.

Comment (craig-ludington) [reply ethjs/build.boot]: Thanks for reviewing this -- it may be a couple of weeks, but I'll fix everything you pointed out.

**Comment by Deraen** [comment]
Closing as inactive. Please reopen if you update this.

**Pull request closed without merging**